cvmfs/history.h

```diff
@@ -165,6 +165,12 @@ class History {
    */
   virtual bool GetHashes(std::vector<shash::Any> *hashes) const = 0;
 
+  // database file management controls
+  virtual void TakeDatabaseFileOwnership() {};
```

Comment:
Not sure if there will be another implementation of `History` soon, apart from `SqliteHistory` and `MockHistory`. But I agree, it cannot hurt to [make it pure virtual](https://github.com/reneme/cvmfs/commit/5d26bca7d3ba6f757e36a9f56cfb3c25c375c495).